mmv1/templates/terraform/constants/monitoring_uptime_check_config.go.erb

```diff
@@ -0,0 +1,3 @@
+func resourceMonitoringUptimeCheckConfigHttpCheckPathDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
+	return old == "/"+new
```

Comment:
Will the API always add the `/` ?

Reply:
Only when the user specified path does not start with "/"

Reply:
I think DiffSuppressFunc only works on detected diffs, so if no diffs detected (in this case, if user's config and API return values both start with "/", it should create no diffs) the diffsuppress will not take effect.

We also have other tests with path start with "/", and it seems they all passed the tests https://ci-oss.hashicorp.engineering/buildConfiguration/GoogleCloudBeta_ProviderGoogleCloudBetaMmUpstreamVcr/268585?buildTab=tests&status=all&name=TestAccMonitoringUptimeCheckConfig&pager.currentPage=1

Reply:
That makes a lot of sense, thanks for the clarification!